pkg/export/export.go

```diff
@@ -419,6 +427,11 @@ func (e *Exporter) Export(metadata MetadataFunc, batch []record.RefSample) {
 			if sampleInRange(s.proto, start, end) {
 				e.enqueue(s.hash, s.proto)
 			} else {
+				// Hashed protos should only ever have one point. If this is a distribution
```

Comment:
What do you mean by "Hashed protos"?

Reply:
bb92b3e614cf62d8bf2101c5825134e02c755b9e